### PR TITLE
Debug container environment variables

### LINKED ISSUE #11278: Debug container environment variables
see https://github.com/symfony/symfony/pull/27684

### PR DESCRIPTION
**Pull request by dFayet**
Hi,

This adds a little documentation for the env vars debug introduced by 
 https://github.com/symfony/symfony/pull/27684

Fixes #11278

### CODE REVIEW

File: configuration/environment_variables.rst [hunk 1]
```diff
@@ -788,3 +788,87 @@ configuration::
...
+
+     ----------------- -----------------
+      Default value     n/a
+      Real value        "[1, "2.5", 3]"
+      Processed value   "2.5"
+     ----------------- -----------------
+
+     // Note real values might be different between web and CLI.
+
+.. warning::
```
Comment (noniagriconomie) [inline configuration/environment_variables.rst]: what about info? warning seem to much :)

File: configuration/environment_variables.rst [hunk 2]
```diff
@@ -788,3 +788,87 @@ configuration::
...
+     BAR              null              n/a
+    ---------------- ----------------- ---------------------------------------------
+
+    // Note real values might be different between web and CLI.
+
+If some variables used by the container are not defined, you will see a warning with the list of all the
+missing variables.
+
+You can also filter the variables by their name:
+
```
Comment (noniagriconomie) [inline configuration/environment_variables.rst]: extrat line

File: configuration/environment_variables.rst [hunk 3]
```diff
@@ -788,3 +788,87 @@ configuration::
...
+    ---------------- ----------------- ---------------------------------------------
+     Name             Default value     Real value
+    ---------------- ----------------- ---------------------------------------------
+     APP_SECRET       n/a               "471a62e2d601a8952deb186e44186cb3"
+     FOO              "[1, "2.5", 3]"   n/a
+     BAR              null              n/a
+    ---------------- ----------------- ---------------------------------------------
+
+    // Note real values might be different between web and CLI.
```
Comment (noniagriconomie) [inline configuration/environment_variables.rst]: maybe a note detailing default vs real values?

**Review by noniagriconomie — COMMENTED** [review]
nice addition

File: configuration/environment_variables.rst (hunk 3 above)
Comment (dFayet) [reply configuration/environment_variables.rst]: Is it really necessary as the default processor is described just ahead?

Comment (dFayet) [reply configuration/environment_variables.rst]: I've added a small note for this

**Comment by javiereguiluz** [comment]
@dFayet thanks a lot for this contribution! Sadly, since you contributed this we've changed the docs about configuration a lot, so this is "unmergeable". I tried to get your commit, rebase with the latest changes and make the needed changes ... but my Git skills are not as good, so I failed at doing that. Instead, I created #11865 based on your work.

I'm really sorry and I hope you can make more contributions in the future so we can merge them directly as we do with most of the pull requests. Thanks!

**Pull request closed without merging**